models/comments.models.js

```diff
@@ -32,3 +32,28 @@ exports.insertComment = (comment, article_id) => {
       return insertedComment.rows[0];
     });
 };
+
+exports.checkCommentExists = (comment_id) => {
+  return db
+    .query(
+      `SELECT comment_id FROM comments WHERE comment_id=$1;`,
```

Comment:
alternatively we could check the rowCount on the db query result from DELETE to see if a row has been affected 🧠